stable/prometheus/templates/server-scrape-secrets.yaml

```diff
@@ -0,0 +1,10 @@
+{{- range $name, $data := .Values.scrapeSecrets }}
```

Comment:
What happens if scrapeSecrets is not defined, will it behave sensibly?
Have you tested running in the default chart?

Reply:
I have tested it. It will not create the secrets and Prometheus is deployed properly.